### PR TITLE
edit the brew install command

### PR DESCRIPTION
**Pull request by Okeibunor**
`brew cask` is no longer a `brew` command. 
Edit the code to use `brew <command> --cask` instead.

### CODE REVIEW

File: src/posts/shipping-react-native-apps-with-fastlane.md [hunk 1]
```diff
@@ -19,7 +19,7 @@ That's when [**Fastlane**](https://fastlane.tools) comes to the rescue ⛑! In t
 Before [installing Fastlane](https://github.com/fastlane/fastlane#installation) make sure you have the latest Xcode command line tools installed, then install Fastlane ⬇️.
 
 ```bash
-$ brew cask install fastlane
+$ brew install fastlane --cask
```
Comment (carloscuesta) [inline src/posts/shipping-react-native-apps-with-fastlane.md:22]: Hey! According to the official docs `--cask` is not needed anymore, right?

See: https://docs.fastlane.tools

CleanShot 2023-09-16 at 20 25 42@2x

Comment (Okeibunor) [reply src/posts/shipping-react-native-apps-with-fastlane.md]: that's right actually. I've corrected accordingly.

**Review by carloscuesta — APPROVED** [review]
LGTM! Thanks for raising a PR

**Comment by Okeibunor** [comment]
> LGTM! Thanks for raising a PR 

You're welcome!

**Pull request merged**